atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/async/CqlQuerySpec.java

```diff
@@ -31,5 +33,9 @@
 
     String formatQueryString();
 
+    ConsistencyLevel queryConsistency();
```

Comment:
Renamed `bind` to `makeExecutableStatement` and moved `setConsistencyLevel` and `preparedStatement.bind()` inside.

Reply:
I believe I changed my mind in person, I think you want to be as explicit as you were originally doing, so you can't "forget" to set the consistency level. You probably want to not put it in the builder, but still expose it as part of the `QuerySpec`, so that the overall runner will set it. i.e. override the method inside `GetQS` but don't put any of the immutables stuff on it.